scripts/lib/libvirt/compute-config

```diff
@@ -34,7 +35,10 @@ def main():
 
     args = parser.parse_args()
 
-    print(libvirt_setup.compute_config(args))
+    machine="pc-0.14"
+    if os.system("rpm -q qemu|grep -q qemu-2")==0:
```

Comment:
I dont expect much more rpm to be used and I dont see it would not get simpler than this.
Also our rpm-python package seems to come without documentation and http://rpm5.org/docs/api/classRpm.html is not much better.